src/components/breadcrumbs/breadcrumbs.yaml

```diff
@@ -1,3 +1,30 @@
+params:
+- name: items
+  type: array
+  required: true
+  description: Array of breadcrumbs item objects.
+  params:
+  - name: text
+    type: string
+    required: true
+    description: If `html` is set, this is not required. Text to use within the breadcrumbs item. If `html` is provided, the `text` argument will be ignored.
+  - name: html
+    type: string
+    required: true
```

Comment:
did we postpone the 'what happens when text OR html are required?

Reply:
Good question, I've included that in the PR description. We decided with @joelanman to fall back using the description of the attributes to explain the relationship.